src/components/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import {
+  CardText, CardBody, CardSubtitle, FormGroup,
+} from 'reactstrap';
+import { connect } from 'react-redux';
+import propTypes from 'prop-types';
+import './resetpassword.scss';
+import * as actions from '../../../store/actions/authActions/resetPassword';
+import Card from '../card/card';
+import Button from '../Button';
+import Input from '../Inputs/Input';
+import Footer from '../Footer/Footer';
+import { Header } from '../Header/Header';
+import { Heading } from '../Heading/Heading';
+/**
+ * @description
+ * @param
+ * @returns
+ */
+export class ResetPasswordCard extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleEmail = this.handleEmail.bind(this);
+    this.state = {
+      email: '',
+    };
+  }
+
+  handleEmail(e) {
+    const { value } = e.target;
+    this.setState({ email: value });
+  }
+
+  handleSubmit() {
+    const { ResetPassword } = this.props;
+    const { history } = this.props;
+    const { email } = this.state;
+    ResetPassword(email, history);
+  }
+
+  render() {
+    const { passwordResetSuccess, passwordResetError, loading } = this.props;
+    return (
+      <div className="containers">
+        <div className="bg-image" />
+        <Header />
+        <Card>
+          <CardBody>
+            <CardSubtitle>
+              <i className="fas fa-lock" />
```

Comment:
@beejay1293 Kindly refer to [https://github.com/FortAwesome/react-fontawesome](url) on how to use font-awesome icons with React. There is a library of icons created in the base project found at `src/index` where you can add the icons you need and use is in your component